docs/getting-started/concepts/permission.md

```diff
@@ -0,0 +1,143 @@
+
+## Concepts
+
+The permission model is based on the following components:
+- A `resource` is a Feast object that we want to secure against unauthorized access.
+  - We assume that the resource has a `name` attribute and optional dictionary of associated key-value `tags`.
+- An `action` is a logical operation executed on the secured resource, like:
+  - `create`: Create an instance.
+  - `read`: Access the instance state.
+  - `update`: Update the instance state.
+  - `delete`: Delete an instance.
+  - `query`:  Query both online and offline stores.
+  - `query_online`:  Query the online store.
+  - `query_offline`:  Query the o ffline store.
+  - `write`:  Query on any store.
+  - `write_online`:  Write to the online store.
+  - `write_offline`:  Write to the offline store.
+- A `policy` identifies the rule for enforcing authorization decisions on secured resources, based on the current user.
+  - A default implementation is provided for role-based policies, using the user roles to grant or deny access to the requested actions
+  on the secured resources.
+
+The `Permission` class identifies a single permission configured on the feature store and is identified by these attributes:
+- `name`: The permission name.
+- `types`: The list of protected resource  types. Defaults to all managed types, e.g. the `ALL_RESOURCE_TYPES` alias
+- `with_subclasses`: Specify if sub-classes are included in the resource match or not. Defaults to `True`.
+- `name_pattern`: A regex to match the resource name. Defaults to `None`, meaning that no name filtering is applied
+- `required_tags`: Dictionary of key-value pairs that must match the resource tags. Defaults to `None`, meaning that no tags filtering is applied.
+- `actions`: The actions authorized by this permission. Defaults to `ALL_VALUES`, an alias defined in the `action` module.
+- `policy`: The policy to be applied to validate a client request.
+
+Given the above definitions, the feature store can be configured with granular control over each resource, enabling partitioned access by 
+teams to meet organizational requirements for service and data sharing, and protection of sensitive information.
+
+The `feast` CLI includes a new `permissions` command to list the registered permissions, with options to identify the matching resources for each configured permission and the existing resources that are not covered by any permission.
+
+{% hint style="info" %}
+**Note**: Feast resources that do not match any of the configured permissions are not secured by any authorization policy, meaning any user can execute any action on such resources.
+{% endhint %}
+
+## Configuration examples
+This permission configuration allows to access the resource state and query all the stores for any feature view or feature service
+to all users with role `super-reader`:
+```py
+Permission(
+    name="feature-reader",
+    types=[FeatureView, FeatureService],
```

Comment:
Is this referring to a an instantiation of the FeatureView class? 

I would recommend creating a full [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

Reply:
ATM we've prepared a reproducible example including a fully distributed environment, in the `examples/rbac` folder, working on a K8s cluster with detailed instructions. 
To support users who are not familiar with K8s, we can prepare another example running all the servers from the local environment and using an OIDC authorization for which we can spin a Keycloak cluster. This is similar to what we did in our POC some time ago.

Would that be enough?

Reply:
Yeah I think so!

Reply:
@franciscojavierarceo pls TAL [here](https://github.com/feast-dev/feast/tree/4e778a7f9c755f9b3e0dbae932f199916426a78f/examples/rbac-local), an example driven by notebooks to build a fully operational, distributed system with OIDC authorization.